### ISSUE TITLE
Simulator luminosity ratio is less than 4.5:1 on high contrast on mode

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-microbit#800]
**Repro steps:**
• Navigate to https://makecode.microbit.org/beta
• Click "More..." -> "High Contrast On" to open high contrast mode
• Verify the luminosity ratio for the simulator

**Expect result:**
 Luminosity ratio should be more than 4.5:1
**Actual result:**
 Simulator luminosity ratio is less than 4.5:1 on high contrast on mode
![1](https://user-images.githubusercontent.com/23466737/40348266-bddd7e20-5d92-11e8-940a-fc2aafadb137.PNG)

**Comment by jwunderl** [comment]
My understanding is that this graphic falls under the category of both being a 'logo' and an 'incidental graphic element', so it is exempt from the contrast requirements / not a required accessibility issue.

Could fix this by making the color always be the same for HC mode though if preferred?
